examples/make-db-migration.rs

```diff
@@ -122,45 +120,53 @@ fn shielded_reward_precision_migration() {
             force: false,
         });
     }
+}
 
-    let changes = migrations::DbChanges {
-        changes: updates.into_iter().collect(),
-    };
-    std::fs::write(
-        "shielded_reward_precision_migration.json",
-        serde_json::to_string(&changes).unwrap(),
-    )
-    .unwrap();
+// A convenience data structure to allow token addresses to be more readably
+// expressed as a channel ID and base token instead of a raw Namada address.
+pub enum TokenAddress {
+    // Specify an IBC address. This can also be done more directly using the
+    // Self::Address variant.
+    Ibc(ChannelId, BaseToken),
+    // Directly specify a Namada address
+    Address(Address),
 }
 
 // Demonstrate clearing MASP rewards for the given IBC tokens by overwriting
 // their allowed conversions with conversions that do not contain rewards.
-fn shielded_reward_reset_migration() {
-    pub type ChannelId = &'static str;
-    pub type BaseToken = &'static str;
-
+fn shielded_reward_reset_migration(
+    updates: &mut Vec<migrations::DbUpdateType>,
+) {
+    // The address of the native token. This is what rewards are denominated in.
+    let native_token =
+        Address::from_str("tnam1qxgfw7myv4dh0qna4hq0xdg6lx77fzl7dcem8h7e")
```

Comment:
Can we make the string here a const up above?

Reply:
Put the string into a `const` above, thanks!